mcs/class/System.ServiceModel/Makefile

```diff
@@ -32,6 +38,10 @@ LIB_MCS_FLAGS += -define:HAS_ACTIVATION -r:System.ServiceModel.Activation.dll
 endif
 endif
 
+endif
```

Comment:
This change is really hard to read, we're adding 2 extra levels of nesting in the Makefile.